plugins/grpc-transport/build.gradle

```diff
@@ -74,4 +74,14 @@ dependencies {
     compile 'io.grpc:grpc-protobuf'
     compile 'io.grpc:grpc-netty'
     compile 'io.grpc:grpc-services'
+
+    testCompileOnly 'org.projectlombok:lombok'
+    testAnnotationProcessor 'org.projectlombok:lombok'
+    testCompile 'org.junit.jupiter:junit-jupiter-api'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
```

Comment:
isn't that better to move on top-level and define for all subprojects?

Reply:
not until all subprojects use jupiterr